perun-auditlogger/src/main/java/cz/metacentrum/perun/auditlogger/logger/impl/EventLoggerImpl.java

```diff
@@ -81,20 +81,21 @@ public void run() {
 			while (running) {
 
 				int sleepTime = 1000;
+				log.debug("Waiting for audit messages.");
```

Comment:
Regarding my other comment, this logging should be probably moved into inner `while` cycle.